x-pack/legacy/plugins/siem/server/lib/detection_engine/alerts/signals_alert_type.ts

```diff
@@ -15,68 +16,70 @@ import { buildEventsReIndex } from './build_events_reindex';
 // once scrolling and other things are done with it.
 // import { buildEventsQuery } from './build_events_query';
 
-export const signalsAlertType: AlertType = {
-  id: `${APP_ID}.signals`,
-  name: 'SIEM Signals',
-  actionGroups: ['default'],
-  async executor({ services, params, state }: AlertExecutorOptions) {
-    // TODO: We need to swap out this arbitrary number of siem-signal id for an injected
-    // data driven instance id through passed in parameters.
-    const instance = services.alertInstanceFactory('siem-signals');
+export function getAlertType({ logger }: { logger: Logger }): AlertType {
```

Comment:
In our project we typically try to use the fat arrow functions as a style and keep the names of the functions the same as the file when there is only one within the file.

For example this would like this:
```ts
export const signalsAlertType = ({ logger }: { logger: Logger } ): AlertType {
```

Reply:
Not a problem, thanks for letting me know. The fix for this can be found in this commit: https://github.com/elastic/kibana/pull/46954/commits/9741654daca56c197407f1cbe56ad6445c6bc42b.